src/app/score/score.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+import { Observable, of } from 'rxjs';
+
+@Component({
+  selector: 'app-score',
+  templateUrl: './score.component.html',
+  styleUrls: ['./score.component.scss'],
+})
+export class ScoreComponent {
+  winner$: Observable<string> = of('circle');
```

Comment:
Shouldn't it be Subject - to be able to control it?

Reply:
I will make game control service, with all game logic and use there behavior subject